src/common/Experiments/Experiments-Live/Experiments.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Experimentation
+{
+    using System.Text.Json;
+    using Microsoft.PowerToys.Settings.UI.Library.Telemetry.Events;
+    using Microsoft.PowerToys.Telemetry;
+    using Microsoft.VariantAssignment.Client;
+    using Microsoft.VariantAssignment.Contract;
+
+    public class Experiments
+    {
+        private enum FeatureFlag
+        {
+            Current,
+            Alternate,
+        }
+
+        public bool EnableLandingPageExperiment()
```

Comment:
@crutkas, does PowerToys already have established patterns for performance telemetry? Understanding typical time to assign experiment state might inform how we use experiments in initialization / first run experiences in the future.

Reply:
PowerToys Run has some of this to measure boot time. But I believe this is the only instance.
Is this something that can be calculated server-side with the current telemetry by comparing events?